### PR TITLE
[FEATURE] Add automated execution test.

### PR DESCRIPTION
**Pull request by Kuzirashi**

### CODE REVIEW

File: src/Actions/Actions.ts [hunk 1]
```diff
@@ -88,23 +94,29 @@ export default class Actions {
       value: 0,
       gas: gasToExecute,
       // TODO estimate gas above
```
Comment (Bagaric) [inline src/Actions/Actions.ts]: You can also remove this comment with it.

File: test/TestScheduleTx.ts [hunk 2]
```diff
@@ -44,6 +44,10 @@ export const getHelperMethods = (web3 : any) => {
   return { waitUntilBlock };
 };
 
+export const SCHEDULED_TX_PARAMS = {
+  callValue: new BigNumber(Math.pow(10, 18))
```
Comment (Bagaric) [inline test/TestScheduleTx.ts]: Might be better to use `web3.eth.toWei(10, 'ether')` which generates a BigNumber.

Comment (Kuzirashi) [reply test/TestScheduleTx.ts]: Actually `web3.toWei(10, 'ether')` returns a string.

Comment (Bagaric) [reply test/TestScheduleTx.ts]: It returns a string if you provide it with a string. Gives a BigNumber if you provide it with a BigNumber. https://github.com/ethereum/wiki/wiki/JavaScript-API#returns-16

**Pull request merged**